magento/magento2
# Implement Tax Rate Mass Delete feature

### Description (*)
Implemented MassDelete for the Tax Rates Grid
![1626271525599](https://user-images.githubusercontent.com/41998275/125636218-cf253e3a-7b34-4cd5-87de-e05c55ffcc39.jpg)

**Note**
`app/code/Magento/Tax/Model/TaxRateCollection.php::getColumnValue` method was modified, because colection was preloaded and the `getColumnValue` method only returns the values from this collection, and the select_all checkbox only selected values from the current pages. This modification helps to load all ids to select_all options for MassDelete.

### Fixed Issues (if relevant)

1. Fixes magento/magento2#33399

### Manual testing scenarios (*)
1. GO to the Tax Zones and Rates page
2. Select items
![1626271550154](https://user-images.githubusercontent.com/41998275/125636345-99923013-5279-487d-b37a-6d2087a4cbe7.jpg)
3.  Select Delete Items and click Submit
4. Confirm Delete
![1626271577815](https://user-images.githubusercontent.com/41998275/125636421-c8b56e0a-36cc-4006-8f02-033e0e203bff.jpg)
5. See Success message
![1626271638523](https://user-images.githubusercontent.com/41998275/125636498-7540503a-874d-4063-8d65-60da52280645.jpg)




### Contribution checklist (*)
